eth-contracts/contracts/DelegateManager.sol

```diff
@@ -361,8 +357,11 @@ contract DelegateManager is RegistryContract {
             totalBalanceInSPFactory.add(spDelegateInfo[msg.sender].totalDelegatedStake)
         );
 
-        // Require claim availability
-        require(totalBalanceInStaking > totalBalanceOutsideStaking, "No stake available to claim");
+        // No-op if balance is already equivalent
+        // This case can occur if no rewards due to bound violation or all stake is locked
```

Comment:
so basically if there's no change in tokens don't bother redistributing cause the distriibution didn't change?

Reply:
that's right